controller_manager/src/controller_manager.cpp

```diff
@@ -1221,4 +1232,15 @@ void ControllerManager::RTControllerListWrapper::wait_until_rt_not_using(
   }
 }
 
+int ControllerManager::get_update_rate() const 
+{
+  return update_rate_;
+}
+
+void ControllerManager::configure() {
```

Comment:
I don't see much purpose in adding this additional method.

Reply:
It somewhat mirrors the controller setup, that we read parameters in the configure stage. Where else would you want to move parsing this to then?

Reply:
I the constructor where other parameters are read.